miso-web/src/main/webapp/pages/editPool.jsp

```diff
@@ -209,6 +209,13 @@
   jQuery(document).ready(function () {
     // Attach Parsley form validator
     Validate.attachParsley('#pool-form');
+
+    // store duplicate indices sequences
+    Pool.duplicateIndices = ${pool.hasDuplicateIndices()};
+    Pool.duplicateIndicesSequences = [];
```

Comment:
That would be nicer, but it looks like the printer backend is the only place we have the `ObjectMapper` inside our model objects. Is that happenstance, or a deliberate choice?